cmd/node/config/genesisSmartContracts.json

```diff
@@ -3,7 +3,7 @@
     "owner": "erd1vxy22x0fj4zv6hktmydg8vpfh6euv02cz4yg0aaws6rrad5a5awqgqky80",
     "filename": "./config/genesisContracts/delegation.wasm",
     "vm-type": "0500",
-    "init-parameters": "%auction_sc_address%@03E8@00@030D40@030D40",
+    "init-parameters": "%validator_sc_address%@03E8@00@030D40@030D40",
```

Comment:
is this backwards compatible? does it need changes in the delegation.wasm?

Reply:
no need to change in delegation.wasm. the address is the same.

Reply:
ok